Add simple runbooks for all enabled alerts

simu opened the pull request:
<!--
Thank you for your pull request. Please provide a description above and
review the checklist below.

Contributors guide: ./CONTRIBUTING.md
-->

## Checklist
<!--
Remove items that do not apply. For completed items, change [ ] to [x].
-->

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog
- [x] Link this PR to related issues.

<!--
NOTE: these things are not required to open a PR and can be done afterwards,
while the PR is open.
-->

simu commented:
Note: some alerts don't have debugging steps, because we don't have any experience with them yet, or because we're going to link to the runbooks for replacing a disk/storage node instead.

corvus-ch commented on component/alertrules.libsonnet:
```diff
@@ -64,6 +64,9 @@ local patch_alerts = {
+local runbook(alertname) =
+  'https://hub.syn.tools/rook-ceph/runbooks/%s.html' % alertname;
+
 /* FROM HERE: should be provided as library function by
```
Is/will be there an ticket to have this converted to a library?

simu replied:
https://github.com/appuio/component-openshift4-monitoring/issues/36

corvus-ch commented on docs/modules/ROOT/pages/runbooks/CephOSDSlowOps.adoc:
```diff
@@ -0,0 +1,29 @@
+
+== icon:glasses[] Overview
+
+This alert fires if some Ceph OSD requests take too long to process.
```
Whats the concequence of this? Do IO on volumes drop or something else?

simu replied:
Expanded the description for the alert

corvus-ch commented on docs/modules/ROOT/pages/runbooks/CephPGRepairTakingTooLong.adoc:
```diff
@@ -0,0 +1,29 @@
+
+== icon:glasses[] Overview
+
+This alert fires when Ceph takes longer than one hour to repair a inconsistent placement group (PG).
```
```suggestion
This alert fires when Ceph takes longer than one hour to repair an inconsistent placement group (PG).
```

corvus-ch reviewed (changes requested):
I realy like what you have done with all those partials. It might be worth to add another one that gets included into all runbooks: "If you gain knowledge beyond what is written here, please update" or something along that line.

Also, what is the deal with all those `// Add detailed steps to debug and resolve the issue` lines?

simu commented:
> I realy like what you have done with all those partials. It might be worth to add another one that gets included into all runbooks: "If you gain knowledge beyond what is written here, please update" or something along that line.

Added a note on all pages in the latest version.

> Also, what is the deal with all those `// Add detailed steps to debug and resolve the issue` lines?

The comments are part of the alert runbook template in `docs/modules/ROOT/pages/runbooks/_RuleTemplate.adoc`. I've now added a how-to describing how a new alert runbook can be generated from the template.
